modules/back-end/src/Domain/SemanticPatch/FlagInstructionKind.cs

```diff
@@ -28,8 +28,10 @@ public static class FlagInstructionKind
 
     public const string UpdateDisabledVariation = nameof(UpdateDisabledVariation);
 
-    public const string UpdateDefaultVariation = nameof(UpdateDefaultVariation);
+    public const string UpdateDefaultRuleVariationOrRollouts = nameof(UpdateDefaultRuleVariationOrRollouts);
```

Comment:
Any reason to change this name ?

Reply:
This is to keep the name consistant with others.
ex: UpdateRuleVariationOrRollouts